src/electrai/dataloader/mp.py

```diff
@@ -127,8 +127,8 @@ def __getitem__(self, idx: int):
         label = self.read_data(self.data[idx][1])
 
         if self.rho_type == "chgcar":
-            data = data.data["total"] / np.prod(data.data["total"].shape)
-            label = label.data["total"] / np.prod(label.data["total"].shape)
```

Comment:
I see that https://github.com/jkitchin/vasp was archived, but is there another vasp library we could use, so that we're not implementing all of the normalization ourself? Does pymatgen have anything for this?

I ask because while this seems fine, it would be nice to have a standard implementation so that if the chgcar file structure changes or there are other improvements, we don't have to reimplement the density calculation ourselves.